app/components/ui/forms.tsx

```diff
@@ -0,0 +1,14 @@
+import type { InputHTMLAttributes, ReactNode } from 'react';
+
+type InputProps = InputHTMLAttributes<HTMLInputElement> & {
+  label: ReactNode;
+};
+
+export function Input({ label, ...props }: InputProps) {
+  return (
+    <label className="flex w-full">
+      <span className="sr-only">{label}</span>
```

Comment:
That is a really great idea to make the Input field reusable and put it inside the label which is for screen readers!